sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md

```diff
@@ -4,12 +4,20 @@
 
 ### Features Added
 
+- Updated token internal validation to utlizie Microsoft.IdentityModel.Protocols.OpenIdConnect to get OIDC configuration.
+
 ### Breaking Changes
 
+- Removing 'TeantId' from the AuthenticationEventTrigger Attribute API configuration
+- Adding 'AuthorizedPartyAppId', 'AuthorityUrl' to the AuthenticationEventTrigger Attribute API configuration
+- Adding 'AuthenticationEvents__ShowPIIDataInLogsKey` (defaults to true) to configuration manager for logging PIIData in logs
```

Comment:
In Azure.Identity, we named this [IsUnsafeSupportLoggingEnabled](https://github.com/Azure/azure-sdk-for-net/blob/1c4302c970516a32295bc6b06678ac2ec962cf68/sdk/identity/Azure.Identity/src/Credentials/TokenCredentialOptions.cs#L40) since PII is an outdated term, and this makes it more clear that it could log sensitive information.